### ISSUE TITLE
Failing towards the end of the build - Forem

### ISSUE THREAD

**Issue by pureflo** [akhil-naidu/t2d#16]
Followed your tutorial for Forem however this is the final steps within the script and seems to fail.

```
> -----> Attempting pre-flight checks (web.1)
       Waiting for 10 seconds ...
 !     App container failed to start!!
=====> nforem web container output:
       buildpack=pgbouncer at=pgbouncer-enabled
       buildpack=pgbouncer at=config-gen-start
       Setting DATABASE_URL_PGBOUNCER config var
       buildpack=pgbouncer at=config-gen-end
       buildpack=pgbouncer at=config-gen-override DATABASE_URL
       buildpack=pgbouncer at=pgbouncer-launched pid=246 signal=SIGINT
       buildpack=pgbouncer at=pgbouncer-start
       buildpack=pgbouncer at=app-launched pid=247
       buildpack=pgbouncer at=app-start
       2022-07-10 03:47:40.681 UTC [248] LOG kernel file descriptor limit: 1048576 (hard: 1048576); max_client_conn: 100, max expected fd use: 114
       2022-07-10 03:47:40.682 UTC [248] LOG listening on 127.0.0.1:6000
       2022-07-10 03:47:40.683 UTC [248] LOG listening on unix:/tmp/.s.PGSQL.6000
       2022-07-10 03:47:40.683 UTC [248] LOG process up: PgBouncer 1.14.0, libevent 2.1.8-stable (epoll), adns: evdns2, tls: OpenSSL 1.1.1  11 Sep 2018
       [251] Puma starting in cluster mode...
       [251] * Puma version: 5.6.4 (ruby 3.0.2-p107) ("Birdie's Version")
       [251] *  Min threads: 5
       [251] *  Max threads: 5
       [251] *  Environment: production
       [251] *   Master PID: 251
       [251] *      Workers: 2
       [251] *     Restarts: (✔) hot (✖) phased
       [251] * Preloading application
       2022-07-10 03:47:50.358 UTC [248] LOG C-0x7fb9edce8650: db1/postgres@127.0.0.1:35066 login attempt: db=db1 user=postgres tls=no
       2022-07-10 03:47:50.359 UTC [248] LOG S-0x7fb9edd294d0: db1/postgres@172.17.0.2:5432 new connection to server (from 172.17.0.4:35848)
       2022-07-10 03:47:50.374 UTC [248] LOG S-0x7fb9edd294d0: db1/postgres@172.17.0.2:5432 SSL established: TLSv1.3/TLS_AES_256_GCM_SHA384/ECDH=prime256v1
       2022-07-10 03:47:50.375 UTC [248] ERROR S-0x7fb9edd294d0: db1/postgres@172.17.0.2:5432 cannot do SCRAM authentication: wrong password type
       2022-07-10 03:47:50.375 UTC [248] LOG C-0x7fb9edce8650: db1/postgres@127.0.0.1:35066 closing because: server login failed: wrong password type (age=0s)
       2022-07-10 03:47:50.376 UTC [248] WARNING C-0x7fb9edce8650: db1/postgres@127.0.0.1:35066 pooler error: server login failed: wrong password type
       2022-07-10 03:47:50.376 UTC [248] LOG S-0x7fb9edd294d0: db1/postgres@172.17.0.2:5432 closing because: failed to answer authreq (age=0s)
       [251] ! Unable to load application: ActiveRecord::ConnectionNotEstablished: connection to server at "127.0.0.1", port 6000 failed: ERROR:  server login failed: wrong password type
       bundler: failed to load command: puma (/app/vendor/bundle/ruby/3.0.0/bin/puma)
       /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/postgresql_adapter.rb:87:in `rescue in new_client': connection to server at "127.0.0.1", port 6000 failed: ERROR:  server login failed: wrong password type (ActiveRecord::ConnectionNotEstablished)
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:656:in `public_send'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:656:in `new_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:700:in `checkout_new_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:679:in `try_to_checkout_new_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:640:in `acquire_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:341:in `checkout'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:181:in `connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_handler.rb:211:in `retrieve_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_handling.rb:313:in `retrieve_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_handling.rb:280:in `connection'
       	from /app/config/initializers/hypershield.rb:11:in `<main>'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:39:in `load'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:39:in `load'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/engine.rb:667:in `block in load_config_initializer'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activesupport-7.0.2.4/lib/active_support/notifications.rb:208:in `instrument'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/engine.rb:666:in `load_config_initializer'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/engine.rb:620:in `block (2 levels) in <class:Engine>'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/engine.rb:619:in `each'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/engine.rb:619:in `block in <class:Engine>'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/initializable.rb:32:in `instance_exec'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/initializable.rb:32:in `run'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/initializable.rb:61:in `block in run_initializers'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:228:in `block in tsort_each'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:422:in `block (2 levels) in each_strongly_connected_component_from'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:431:in `each_strongly_connected_component_from'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:421:in `block in each_strongly_connected_component_from'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/initializable.rb:50:in `each'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/initializable.rb:50:in `tsort_each_child'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:415:in `call'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:415:in `each_strongly_connected_component_from'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:349:in `block in each_strongly_connected_component'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:347:in `each'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:347:in `call'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:347:in `each_strongly_connected_component'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:226:in `tsort_each'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:205:in `tsort_each'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/initializable.rb:60:in `run_initializers'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/application.rb:372:in `initialize!'
       	from /app/config/environment.rb:5:in `<top (required)>'
       	from config.ru:3:in `require_relative'
       	from config.ru:3:in `block in <main>'
       	from /app/vendor/bundle/ruby/3.0.0/gems/rack-2.2.4/lib/rack/builder.rb:116:in `eval'
       	from /app/vendor/bundle/ruby/3.0.0/gems/rack-2.2.4/lib/rack/builder.rb:116:in `new_from_string'
       	from /app/vendor/bundle/ruby/3.0.0/gems/rack-2.2.4/lib/rack/builder.rb:105:in `load_file'
       	from /app/vendor/bundle/ruby/3.0.0/gems/rack-2.2.4/lib/rack/builder.rb:66:in `parse_file'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/lib/puma/configuration.rb:348:in `load_rackup'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/lib/puma/configuration.rb:270:in `app'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/lib/puma/runner.rb:150:in `load_and_bind'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/lib/puma/cluster.rb:357:in `run'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/lib/puma/launcher.rb:182:in `run'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/lib/puma/cli.rb:81:in `run'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/bin/puma:10:in `<top (required)>'
       	from /app/vendor/bundle/ruby/3.0.0/bin/puma:23:in `load'
       	from /app/vendor/bundle/ruby/3.0.0/bin/puma:23:in `<top (required)>'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/cli/exec.rb:58:in `load'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/cli/exec.rb:58:in `kernel_load'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/cli/exec.rb:23:in `run'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/cli.rb:483:in `exec'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/cli.rb:31:in `dispatch'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/cli.rb:25:in `start'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/exe/bundle:48:in `block in <top (required)>'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/friendly_errors.rb:103:in `with_friendly_errors'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/exe/bundle:36:in `<top (required)>'
       	from /app/bin/bundle:3:in `load'
       	from /app/bin/bundle:3:in `<main>'
       /app/vendor/bundle/ruby/3.0.0/gems/pg-1.4.1/lib/pg/connection.rb:598:in `async_connect_or_reset': connection to server at "127.0.0.1", port 6000 failed: ERROR:  server login failed: wrong password type (PG::ConnectionBad)
       	from /app/vendor/bundle/ruby/3.0.0/gems/pg-1.4.1/lib/pg/connection.rb:750:in `connect_internal'
       	from /app/vendor/bundle/ruby/3.0.0/gems/pg-1.4.1/lib/pg/connection.rb:726:in `block (2 levels) in connect_to_hosts'
       	from /app/vendor/bundle/ruby/3.0.0/gems/pg-1.4.1/lib/pg/connection.rb:724:in `each'
       	from /app/vendor/bundle/ruby/3.0.0/gems/pg-1.4.1/lib/pg/connection.rb:724:in `block in connect_to_hosts'
       	from /app/vendor/bundle/ruby/3.0.0/gems/pg-1.4.1/lib/pg/connection.rb:710:in `each'
       	from /app/vendor/bundle/ruby/3.0.0/gems/pg-1.4.1/lib/pg/connection.rb:710:in `each_with_index'
       	from /app/vendor/bundle/ruby/3.0.0/gems/pg-1.4.1/lib/pg/connection.rb:710:in `connect_to_hosts'
       	from /app/vendor/bundle/ruby/3.0.0/gems/pg-1.4.1/lib/pg/connection.rb:661:in `new'
       	from /app/vendor/bundle/ruby/3.0.0/gems/pg-1.4.1/lib/pg.rb:69:in `connect'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/postgresql_adapter.rb:78:in `new_client'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:656:in `public_send'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:656:in `new_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:700:in `checkout_new_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:679:in `try_to_checkout_new_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:640:in `acquire_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:341:in `checkout'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_pool.rb:181:in `connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_adapters/abstract/connection_handler.rb:211:in `retrieve_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_handling.rb:313:in `retrieve_connection'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activerecord-7.0.2.4/lib/active_record/connection_handling.rb:280:in `connection'
       	from /app/config/initializers/hypershield.rb:11:in `<main>'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:39:in `load'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bootsnap-1.12.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:39:in `load'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/engine.rb:667:in `block in load_config_initializer'
       	from /app/vendor/bundle/ruby/3.0.0/gems/activesupport-7.0.2.4/lib/active_support/notifications.rb:208:in `instrument'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/engine.rb:666:in `load_config_initializer'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/engine.rb:620:in `block (2 levels) in <class:Engine>'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/engine.rb:619:in `each'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/engine.rb:619:in `block in <class:Engine>'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/initializable.rb:32:in `instance_exec'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/initializable.rb:32:in `run'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/initializable.rb:61:in `block in run_initializers'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:228:in `block in tsort_each'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:350:in `block (2 levels) in each_strongly_connected_component'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:422:in `block (2 levels) in each_strongly_connected_component_from'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:431:in `each_strongly_connected_component_from'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:421:in `block in each_strongly_connected_component_from'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/initializable.rb:50:in `each'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/initializable.rb:50:in `tsort_each_child'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:415:in `call'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:415:in `each_strongly_connected_component_from'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:349:in `block in each_strongly_connected_component'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:347:in `each'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:347:in `call'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:347:in `each_strongly_connected_component'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:226:in `tsort_each'
       	from /app/vendor/ruby-3.0.2/lib/ruby/3.0.0/tsort.rb:205:in `tsort_each'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/initializable.rb:60:in `run_initializers'
       	from /app/vendor/bundle/ruby/3.0.0/gems/railties-7.0.2.4/lib/rails/application.rb:372:in `initialize!'
       	from /app/config/environment.rb:5:in `<top (required)>'
       	from config.ru:3:in `require_relative'
       	from config.ru:3:in `block in <main>'
       	from /app/vendor/bundle/ruby/3.0.0/gems/rack-2.2.4/lib/rack/builder.rb:116:in `eval'
       	from /app/vendor/bundle/ruby/3.0.0/gems/rack-2.2.4/lib/rack/builder.rb:116:in `new_from_string'
       	from /app/vendor/bundle/ruby/3.0.0/gems/rack-2.2.4/lib/rack/builder.rb:105:in `load_file'
       	from /app/vendor/bundle/ruby/3.0.0/gems/rack-2.2.4/lib/rack/builder.rb:66:in `parse_file'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/lib/puma/configuration.rb:348:in `load_rackup'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/lib/puma/configuration.rb:270:in `app'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/lib/puma/runner.rb:150:in `load_and_bind'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/lib/puma/cluster.rb:357:in `run'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/lib/puma/launcher.rb:182:in `run'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/lib/puma/cli.rb:81:in `run'
       	from /app/vendor/bundle/ruby/3.0.0/gems/puma-5.6.4/bin/puma:10:in `<top (required)>'
       	from /app/vendor/bundle/ruby/3.0.0/bin/puma:23:in `load'
       	from /app/vendor/bundle/ruby/3.0.0/bin/puma:23:in `<top (required)>'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/cli/exec.rb:58:in `load'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/cli/exec.rb:58:in `kernel_load'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/cli/exec.rb:23:in `run'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/cli.rb:483:in `exec'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/cli.rb:31:in `dispatch'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/cli.rb:25:in `start'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/exe/bundle:48:in `block in <top (required)>'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/lib/bundler/friendly_errors.rb:103:in `with_friendly_errors'
       	from /app/vendor/bundle/ruby/3.0.0/gems/bundler-2.3.10/exe/bundle:36:in `<top (required)>'
       	from /app/bin/bundle:3:in `load'
       	from /app/bin/bundle:3:in `<main>'
       buildpack=pgbouncer at=app-end
       buildpack=pgbouncer at=exit process=app
       buildpack=pgbouncer at=kill-app pid=247
       buildpack=pgbouncer at=wait-app pid=247
       buildpack=pgbouncer at=kill-aux name=pgbouncer pid=246 signal=SIGINT
       buildpack=pgbouncer at=pgbouncer-end
       2022-07-10 03:47:50.409 UTC [248] LOG got SIGINT, shutting down
       2022-07-10 03:47:50.409 UTC [248] LOG Got SIGTERM, ignoring!
=====> end nforem web container output
There you go :), Leave a like if you successfully configured your Forem
```

**Comment by Surgbc** [comment]
I experienced the same problem, though I can't quite remember exactly where it was failing, when I was trying on a server that had other things already running. I don't quite know what changed. But the build was successful when I tried in a new server.

**Comment by pureflo** [comment]
> I experienced the same problem, though I can't quite remember exactly where it was failing, when I was trying on a server that had other things already running. I don't quite know what changed. But the build was successful when I tried in a new server.

The build I used was done a new Droplet from DigitalOcean so unfortunately that hasn’t changed the outcome for me.

**Comment by Surgbc** [comment]
Your title fails to capture the real problem, and I failed to go through the logs to see that it is the same problem I had in my `new server`. 

Just a technical question? Is that part of the build process? Or the deployment process? I don't understand the technical term to use, but I have outlined a solution for this problem in [this other issue](https://github.com/leewardslope/t2d/issues/17) I created.

In summary, 

1. Solve the DNS issue by changing the change domain pointing to Postgres container to its IP address. This will still leave you with an incompatibility between postgres14.4 in use and the client in nforem.web container. Solve this by:

2. Downgrading postgres to 13.x which is the latest version that allows md5 clients.

**Comment by ShreehariVaasishta** [comment]
Im experiencing the same issue. Not sure how to solve it.
I did try the solution mentioned by @Surgbc in the above comment and have not made any progress yet.
